plugins/network-elements/nuage-vsp/src/com/cloud/network/element/NuageVspElement.java

```diff
@@ -129,6 +145,18 @@
     AgentManager _agentMgr;
     @Inject
     NetworkOfferingDao _ntwkOfferingDao;
+    @Inject
+    ConfigurationDao _configDao;
+    @Inject
+    NuageVspManager _nuageVspManager;
+    @Inject
+    FirewallRulesDao _firewallRulesDao;
+    @Inject
+    FirewallRulesCidrsDao _firewallRulesCidrsDao;
+    @Inject
+    PhysicalNetworkDao _physicalNetworkDao;
+    @Inject
+    NetworkACLItemDao _networkACLItemDao;
 
     @Override
     public boolean applyIps(Network network, List<? extends PublicIpAddress> ipAddress, Set<Service> service) throws ResourceUnavailableException {
```

Comment:
I moved some code here and there, it's shortened now by decomposing into smaller private methods.